pages/api/badges/contract.tsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable @next/next/no-img-element */
+
+/* eslint-disable react/forbid-dom-props */
+import { readFile } from "fs/promises";
+import type { NextApiRequest, NextApiResponse } from "next";
+import { resolve } from "path";
+import satori from "satori";
+import { shortenIfAddress } from "utils/usedapp-external";
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
+  if (req.method !== "POST") {
+    return res.status(400).json({ error: "Please use POST method" });
+  }
+
+  const { address } = JSON.parse(req.body);
+
+  try {
+    const fontBuffer = await readFile(
+      resolve("./public/assets/fonts", "Inter-medium.ttf"),
+    );
```

Comment:
for addresses we use monospace so might want monospace font here? (IBM plex mono...)

Reply:
Following the designs here that uses Inter for both the address and the "Audited" text, I like how it looks